engine/src/main/java/org/hibernate/validator/constraintvalidation/HibernateConstraintValidatorInitializationContext.java

```diff
@@ -39,6 +39,15 @@
 	 */
 	ScriptEvaluator getScriptEvaluatorForLanguage(String languageName);
 
+	/**
+	 * Returns a {@link ScriptEvaluatorFactory} passed at bootstrap.
+	 *
+	 * @return a script evaluator factory. Never null.
+	 *
+	 * @since 6.0.9
+	 */
+	ScriptEvaluatorFactory getScriptEvaluatorFactory();
```

Comment:
This wouldn't need to be exposed if you implement `hashCode()` and `equals()` in the context.

Reply:
Yeah I understand.